Makefile

```diff
@@ -75,7 +75,7 @@ run: ## Run the server inside an IEx shell
 .PHONY: dependencies
 dependencies: ## Install dependencies
 	mix deps.get --force
-	npm install --prefix assets
+	npm ci --prefix assets
```

Comment:
This is great for CI but not for development, the developer should use `npm install`, amd the ci `npm ci`

Reply:
True, I'll see what I could come up with 🤔